src/main/kotlin/it/unibo/lss/fcla/consulting/domain/consulting/events/ConsultingCreatedEvent.kt

```diff
@@ -0,0 +1,22 @@
+package it.unibo.lss.fcla.consulting.domain.consulting.events
+
+import it.unibo.lss.fcla.consulting.domain.consulting.ConsultingId
+import it.unibo.lss.fcla.consulting.domain.consulting.ConsultingType
+import it.unibo.lss.fcla.consulting.domain.consulting.Date
+import it.unibo.lss.fcla.consulting.domain.consulting.MemberId
+import it.unibo.lss.fcla.consulting.domain.contracts.DomainEvent
+import it.unibo.lss.fcla.consulting.domain.freelancer.FreelancerId
+
+/**
+ * @author Stefano Braggion
+ *
+ * Event representing a created consulting
+ */
+data class ConsultingCreatedEvent(
```

Comment:
Maybe ConsultingCreated (without the Event suffix) is better? The same apply for the other classes into the event package

Reply:
it's intentional to have the suffix Event , to distinguish events from commands (probably introduced in the next release)